site/src/sphinx/advanced-zipkin.rst

```diff
@@ -3,4 +3,114 @@
 Zipkin integration
 ==================
 
-TBW - See :api:`BraveService` and :api:`BraveClient`
+If you want to troubleshoot latency problems in microservice architecture, you will want to use distributed
+tracing system such as `Zipkin <https://zipkin.io/>`_. It gathers timing data and shows which component is
+the problem. Armeria supports distributed tracing via `Brave <https://github.com/openzipkin/brave/>`_, which
```

Comment:
which component is the problem -> which component is failing or taking more time than others in a distributed environment.